style.css

```diff
@@ -4,6 +4,7 @@
   min-height: 100vh;
   flex-direction: column;
   background-color: white;
+  background-image: url('/home/ignatius/Documents/Microverse Projects/Portfolio/images/background-image.png');
```

Comment:
- [ ] [REQUIRED] Please implement my and the previous reviewer's suggestions!

1. The content should be aligned with `flexbox`.
   * This gives us a lot of options to align all our elements, plus, it's responsive.
1. The section should have the `margins` (`top`, `bottom`, `left`, and `right`) defined in the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).
   * If it would be `pretty` and `works`, it will be a wonderful website.
1. The `section` should use the correct `font` and `colors` (`background` and `font`) defined for the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).
   * If it's pretty and works it will be a wonderful website.

![image](https://user-images.githubusercontent.com/63915024/212700077-bc76b783-0672-4465-8b27-71c9ac398fe1.png)